flask_restful/reqparse.py

```diff
@@ -133,7 +134,10 @@ def handle_validation_error(self, error):
         :param error: the error that was raised
         """
         help_str = '(%s) ' % self.help if self.help else ''
-        msg = '[%s]: %s%s' % (self.name, help_str, str(error))
+        error_msg = ' '.join([help_str, str(error)]) if help_str else str(error)
+        msg = json.dumps({self.name: "%s" % (error_msg)})
+        if current_app.config.get("BUNDLE_ERRORS", False):
```

Comment:
I think this should (also) be a parameter when creating a new `RequestParser`. Example:

```python
parser = RequestParser(bundle_errors=True)
```

If some arguments are expensive to validate, it might be better to just fail immediately on the first invalid argument (parsing more arguments is not going to make the request succeed after all). The `app.config` value would still be useful as a global default.